### PR TITLE
Bugfix calling static TA

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Bugfixes related to calling static TA from a user TA

### CODE REVIEW

File: core/include/mm/tee_mmu_types.h [hunk 1]
```diff
@@ -55,6 +55,8 @@
 #define TEE_MATTR_CACHE_NONCACHE 0
 #define TEE_MATTR_CACHE_CACHED	1
 
+#define TEE_MATTR_VIRTUAL		(1 << 15)
```
Comment (pascal-brand-st-dev) [inline core/include/mm/tee_mmu_types.h]: `(1 << 13)` instead?

**Comment by pascal-brand-st-dev** [comment]
With or without my comment:
`Reviewed-by: Pascal Brand <pascal.brand@linaro.org>`

File: core/include/mm/tee_mmu_types.h (hunk 1 above)
Comment (jenswi-linaro) [inline core/include/mm/tee_mmu_types.h]: TEE_MATTR_CACHE consumes three bits, the next available bit is this.

**Comment by jenswi-linaro** [comment]
Tag applied

**Pull request merged**